# 2021Workshop-COVID19

## Single cell RNA seq visualization 

According to the United Nations, in the last years, the world has shown an increase in population aging and, as a consequence, chronic age-related diseases. From a national perspective, chronic diseases have severe implications in life expectancy and, as a result, generate economic and political pressures. This year with the COVID-19 pandemic, we have been trying to understand if survivors who have been seriously ill will have some sequelae such as pulmonary fibrosis, an irreversible chronic disease. High-throughput technologies have evolved too fast in the last years, and nowadays, single-cell RNA sequencing has become more popular because of the amount of information that we can obtain measuring thousands of single cells in one experiment. This project is focused on analyzing published single-cell data of COVID-19 and look if we can find some known biomarkers of pulmonary fibrosis. We will try classical visualization tools for single-cell data using R packages, but we will be creative and find new ways of visualizing data for finding patterns. Our main objective would be to guide hypothesis generation through the visualization of single-cell COVID data.

*Goal*: Visualization of single-cell COVID data for hypothesis generation.

  Some objectives are: 

    a) To manipulate single cell data formats and incorporate into R

    b) To learn different packages for analyzing single cell data

    c) To explore different types of visualization for single cell data

    d) To find cell type biomarkers

    e) To compare COVID biomarkers with those from pulmonary fibrosis

    f) To work as bioinformatic team 

    g) To incorporate and exchange information with other teams

    h) To present the results


## Data 

- **Paper:**  (- Authors: Chua, R.L., Lukassen, S., Trump, S., Hennig, B.P., Wendisch, D., Pott, F., Debnath, O., Thürmann, L., Kurth, F., Völker, M.T. and Kazmierski, J., 2020. COVID-19 severity correlates with airway epithelium–immune cell interactions identified by single-cell analysis. Nature biotechnology, 38(8), pp.970-979.)[https://www.nature.com/articles/s41587-020-0602-4]

- **Tissue:** BALF

- **Data type:** scRNA-seq 

- **Samples:** 5 Controls and 19 COVID19 patients

- **Download data:** (Data)[https://figshare.com/articles/dataset/COVID-19_severity_correlates_with_airway_epithelium-immune_cell_interactions_identified_by_single-cell_analysis/12436517] has already been downloaded in the server (kneipe.lavis.unam.mx).

      - **Path:** /home/bana/balf_human_Chua 

- **Raw sequencing:** https://ega-archive.org/datasets/EGAD00001006339/files

------

## Additional data

- **Paper:**  (Wauters, E., Van Mol, P., Garg, A.D., Jansen, S., Van Herck, Y., Vanderbeke, L., Bassez, A., Boeckx, B., Malengier-Devlies, B., Timmerman, A. and Van Brussel, T., 2021. Discriminating mild from critical COVID-19 by innate and adaptive immune single-cell profiling of bronchoalveolar lavages. Cell research, pp.1-19.)[https://www.nature.com/articles/s41422-020-00455-9]

- **Tissue:** BALF

- **Data type:** scRNA-seq and scTCR-seq

- **Samples:** 13 non-COVID19 (with pneumonia) and 22 COVID19 patients

- **Raw sequencing:** https://www.ebi.ac.uk/ega/studies/EGAS00001004717

- **Count matrix:** http://covid19.lambrechtslab.org/



